code/modules/projectiles/guns/magic/wand.dm

```diff
@@ -67,7 +67,8 @@
 	var/message ="<span class='warning'>You irradiate yourself with pure energy! "
 	message += pick("Do not pass go. Do not collect 200 zorkmids.</span>","You feel more confident in your spell casting skills.</span>","You Die...</span>","Do you want your possessions identified?</span>")
 	to_chat(user, message)
-	user.adjustFireLoss(500)
+	user.gib()
```

Comment:
This will, in fact work, as `adjustFireLoss` will properly distribute it over the entirety of the body...or SHOULD.